### PR TITLE
Added support for Daikin AirBase units

### PR DESCRIPTION
**Pull request by fredrike**
**Description:**

pydaikin v1.2.0 adds support for AU AirBase units.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22382

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by klaasnicolaas — APPROVED** [review]
✅ Can be merged after merging the parent PR 👍

**Review by fabaff — APPROVED** [review]

**Pull request merged**
